cl/recap/mergers.py

```diff
@@ -61,6 +61,25 @@
 cnt = CaseNameTweaker()
 
 
+def confirm_docket_number_core_lookup_match(
+    kwargs: dict[str, str | None], docket: Docket, docket_number: str
+) -> Docket | None:
+    """Confirm if the docket_number_core lookup match returns the right docket
+    when pacer_case_id is None, by confirming the docket_number also matches.
+
+    :param kwargs: The lookup kwargs
```

Comment:
It's probably better to pass the actual arguments that we want here. So, instead of passing kwargs, we should pass docket_number_core and pacer_case_id.

Reply:
Yeah, actually considering the suggestion below now it's no needed to pass the `pacer_case_id` neither `docket_number_core`.